app/image_processing.py

```diff
@@ -0,0 +1,109 @@
+import cv2
+import numpy as np
+import re
+from pytesseract import image_to_osd
+from PIL import Image
+from config.config import logger
+
+
+def get_image_orientation(image_path):
+    try:
+        image = Image.open(image_path)
+        osd = image_to_osd(image)
```

Comment:
I prefer the second approach with hough lines or some other approach, osd does not work in most scenarios.